index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+          <meta charset="UTF-8">
+          <meta http-equiv="X-UA-Compatible" content="IE=edge">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+          <link rel="stylesheet" href="css/styles.css">
+          <link rel="stylesheet" href="css/about.css">
+          <title>capstone project</title>
+</head>
+
+<body>
+<main id="main-section">
+<div class="showcase-area">
+          <div>
+          <nav id="top-nav">
+          <ul>
+          <li><a href="#"><i class="bi bi-facebook"></i></a></li>
+          <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+         <li><a href="#">Logout</a></li>
+          </ul>
+          </nav>
+          <header id="main-header">
+          <nav>
+          <div class="flex-column menu">
+         <span class="hamburger"></span>
+         <span class="hamburger"></span>
+          <span class="hamburger"></span>
+          </div>
+           <a href="index.html" class="logo"><img
+                              src="./images/hymusic_logs.png"
+                              alt="our_logo"><span>HYMUSIC</span></a>
+                              <ul class="nav-list hide">
+                              <li id="close"><div class="flex">
+                              <a href="index.html"
+                              id="active"><i class="bi bi-house-fill"></i></a>
+                              <i class="bi bi-x-lg flex"></i>
+                              </div></li>
+                              <li><a href="about.html">About</a></li>
+                              <li><a href="#">Programs</a></li>
+                              <li><a href="#">Register</a></li>
+                              <li><a href="#">Sponsor</a></li>
+                              <li><a href="#">News</a></li>
+                              </ul>
+                              </nav>
+                              </header>
+          <section id="home">
+          <div class="main-sectionA  desktopsecA">
+          <h2 class="head"> "Hello! Sharing World" <br> <br>
+          HYCROWN ENTERTAINMENT CREATIVE COMMON GLOBAL
+          SUMMIT 2022</h2>
+          <p class="para-text paratextA ">
+          A fun festival where people from more than 80
```

Comment:
- [ ] Kindly remove the white space here
![Capturemobblank](https://user-images.githubusercontent.com/81444338/175710469-34b5a316-4647-455c-ae4a-3f8ec5422e6d.PNG)

Reply:
thank you so much i have fix it perhaps the slideshow images not appear but i have fix this  . thank you